package.json

```diff
@@ -7,7 +7,8 @@
     "watch-js": "nodemon -e js --watch app --ignore app/bundle.js -x \"npm run build-bundle-js\"",
     "watch-server": "nodemon server.js",
     "release-build": "npm run build-bundle-js",
-    "debug-build": "parallelshell \"npm run watch-js\" \"npm run watch-server\""
+    "debug-build": "parallelshell \"npm run watch-js\" \"npm run watch-server\"",
+    "heroku-postbuild": "npm run build-bundle-js"
```

Comment:
I think this is Heroku related and can be removed.

Reply:
Deleted heroku-postbuild